Vagrantfile

```diff
@@ -120,6 +120,16 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     vb.cpus = vm_num_cpus
   end
 
+  config.vm.provider "parallels" do |prl, override|
+	# We are using a ubuntu-20 box here since other
+	# boxes which support parallels are outdated and
+	# not popularly downloaded.
+	override.vm.box = "bento/ubuntu-20.04"
```

Comment:
Does `bento/ubuntu-18.04` not work? (Also “20” is not an Ubuntu version; that’s ambiguous between 20.04 and 20.10.)

Reply:
I somehow managed to miss it :(. Updated!